graphql-api/src/cache.ts

```diff
@@ -70,7 +72,11 @@ if (config.REDIS_HOST) {
 export const withCache = (fn: any, keyFn: any, options = {}) => {
   // @ts-expect-error TS(2339) FIXME: Property 'expiration' does not exist on type '{}'.
   //
-  const { expiration = 3600, jsonCacheEnableAll = config.JSON_CACHE_ENABLE_ALL, jsonCacheLargeGenes = config.JSON_CACHE_LARGE_GENES } = options
+  const {
```

Comment:
@sjahl Probably the best way to resolve the type conflicts would be to introduce a type for the options of `withCache`, something like:

```
type WithCacheOptions = {
  expiration?: number
  jsonCacheEnableAll?: string     // or whatever JSON_CACHE_ENABLE_ALL is
  jsonCacheLargeGenes?: string
}
```

And then the start of `withCache` becomes `export const withCache = (fn: any, keyFn: any, options: WithCacheOptions = {}) => { ...`

If you're not pressed for time, this would also be a good juncture to give `fn` and `keyFn` proper types as well, and maybe a name for `fn` that conveys some useful information beyond the fact that it's a function.